ci-operator/step-registry/redhat-performance/benchmark-runner/ibm/redhat-performance-benchmark-runner-ibm-commands.sh

```
@@ -0,0 +1,256 @@
+  elif [[ -s /secret/kubeconfig ]]; then
+    cp /secret/kubeconfig /tmp/kubeconfig
+    export KUBECONFIG=/tmp/kubeconfig
+    echo "Using kubeconfig from Vault fallback"
+  else
+    echo "ERROR: could not fetch kubeconfig" >&2
+    exit 1
+  fi
+
+  # SOCKS proxy through cluster for private DNS resolution
+  SOCKS_PORT=$((RANDOM % 55536 + 10000))
+  ssh ${SSH_ARGS} root@"${CLUSTER_IP}" -fNT -D "${SOCKS_PORT}"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)